### DISCUSSION TITLE
Allow webcam for Jitsi or BigBlueButton webconference

### DISCUSSION THREAD

**Discussion by rschaufe** [SafeExamBrowser/seb-win-refactoring#299] [General]
Normally when you start a webconference Jitsi or BigBlueButton the browser ask for permission to allow the camera. In the browser of SEB this question is not appearing and therefore there is no camera available in the webconference.

Where can you configure that feature? I couldn't find any hint so far.

![image](https://user-images.githubusercontent.com/96474733/146934425-08776d49-d654-43cf-a999-c832a5aed554.png)

**Comment by heinzkrebs** [comment]
My students (100+) access BigBlueButton with SEB without problems, and i don't remember that i have configured a setting for that.

I also was able to access your Jitsi-Server, and the camera get's connected automatically ...

sincerely
  Heinz

**Reply by rschaufe** [reply]
Ok, what version of SEB do you use? I use 3.3.1 Windows?
Thank you.

**Comment by heinzkrebs** [comment]
i tried both now, version 3.3.0 and 3.3.1 on windows 11

**Comment by rschaufe** [comment]
Thank you very much for your answer. So I could turn my attention in the right direction - My test notebook has a hardware problem...

**Comment by dbuechel** [comment]
Access to camera and microphone are automatically granted in the main browser as well as the proctoring feature.
